exercise3.js

```diff
@@ -5,30 +5,37 @@
 // Problem 1:
 // Write a JavaScript function that determines whether a string is empty. If the string is empty, the output should be true,
 // otherwise it should return false. 
-function ifEmpty(str){
-	if (str)
-		return false
-	else
-		return true
-}
+// function ifEmpty(str){
+// 	if (str)
+// 		return false
+// 	else
+// 		return true
+// }
+const isEmpty = string => string.length<1; 
+// console.log(ifEmpty(""))
+console.log(isEmpty);
 
-console.log(ifEmpty(""))
 /************************************************************* */
 
 // Problem 2:
 // Declare a variable that holds a string (e.g., ‘Learning JavaScript is fun!’). Replace a word so that the output 
 //uses the new word instead (e.g., ‘Learning JavaScript is cool!). 
 
-function newStr(str){
-	var newStr = ""
-	var arrs = str.split(" ")
-	for (let i=0; i<arrs.length-1; i++){
-		newStr += arrs[i] + " "
-	}
-	newStr += "cool!"
-	return newStr;
-}
-console.log(newStr("Learning JavaScript is fun!"));
```

Comment:
maybe just split the string into an array, and replace the last element of the array with the replaced word.  that would be a solution that's possibly easier to follow.